model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -91,31 +91,31 @@ data class PackageCurationData(
     fun apply(base: CuratedPackage): CuratedPackage = applyCurationToPackage(base, this)
 }
 
-private fun applyCurationToPackage(base: CuratedPackage, curation: PackageCurationData): CuratedPackage {
-    val pkg = base.pkg
+private fun applyCurationToPackage(targetPackage: CuratedPackage, curation: PackageCurationData): CuratedPackage {
```

Comment:
Commit message: Maybe move "for clarity" into the title or add a dot at the end.

Reply:
Done.